### ISSUE TITLE
LayerY returning wrong coordinate

### ISSUE THREAD

**Issue by kubicle** [waltheri/wgo.js#43]
Not sure yet why and how but I bumped into an issue when adding several boards into a div. 
The Y coordinate of the clicked vertex was returned wrong.
By replacing "layerY" by "offsetY" all now works fine. (offsetX works good, though in my case the X coordinate seemed ok with the current code) 

Is it a known problem? 

I found this article:
http://help.dottoro.com/ljcpdudj.php

The layerX property is rarely useful, use other event properties instead.
To get the position relative to the top-left corner of the document, use the pageX and pageY properties.
If you need the position of the mouse pointer relative to the top-left corner of the browser window's client area, use the clientX and clientY properties.
To get the position relative to the top-left corner of the screen, use the screenX and screenY properties.
Or, if you need the element that is located at a specified position, use the elementFromPoint method.

**Comment by waltheri** [comment]
Hi,
I'd like to know, what is causing the issue. Do you have some example, which breaks with the current code?

**Comment by kubicle** [comment]
I will try to build an example for you. So you confirm that with the current code I should be able to use several boards on the same page without issue?

**Comment by waltheri** [comment]
I think so

**Comment by kubicle** [comment]
I did see a few pixels differences and it seemed offsetY was "more correct" on Chrome.
When ready, I will send you a link to my dev branch with extra logging to show that.
However, I could not reproduce my initial issue that was showing much more than a few pixels (so the stone would end up on the wrong vertex, a bummer). I changed my layout quite a lot since that time so chances are I will not see it again. It is also possible that this bug only happened inside Chrome emulator for Android (quite known to be buggy).

Also, according to same source (link above) offsetX and offsetY are not more recommanded than layerX and layerY. Looks like the right code should use clientX and clientY. If you agree I could investigate more and do a PR. If you don't want to spend time we simply can leave it like this until someone else complains about a similar issue.
